botorch/optim/optimize_homotopy.py

```diff
@@ -155,13 +160,17 @@ def optimize_acqf_homotopy(
         ic_gen_kwargs: Additional keyword arguments passed to function specified by
             `ic_generator`
     """
+    if fixed_features and fixed_features_list:
+        raise ValueError(
+            "Èither `fixed_feature` or `fixed_features_list` can be provided, not both."
+        )
```

Comment:
`fixed_features` as an arg seems redundant if we have `fixed_features_list`. We should probably deprecate the  `fixed_features` arg and raise a warning when passed. And if `fixed_features_list` contains only one element we can just use that with `optimize_acqf`

Reply:
yes, that would be great

Reply:
I implemeted it now for `optimize_acqf_homotopy`, if it is fine for you how it looks there, I will port it to `optimize_acqf_list`.